crates/composer/src/main.rs

```diff
@@ -0,0 +1,37 @@
+use composer::api::Event;
+use composer::api::DEFAULT_SERVER_ADDRESS;
+use eyre::Result;
+use rodio::{source::Source, Decoder, OutputStream};
+use std::fs::File;
+use std::io::BufReader;
+use std::net::UdpSocket;
+
+fn main() -> Result<()> {
+    color_eyre::install()?;
+
+    let socket = UdpSocket::bind(DEFAULT_SERVER_ADDRESS)?;
+    println!("Listening on {}", socket.local_addr()?);
+
+    let (_stream, stream_handle) = OutputStream::try_default()?;
+
+    loop {
+        if let Err(err) = {
+            // Should be more than enough.
+            let mut buf = [0; 1000];
```

Comment:
Network packet size is usually limited by 1500 bytes (minus link, IP and UDP overhead, but that's details), so we could use that:
```suggestion
            // Size up to max normal network packet size should be enough
            let mut buf = [0; 1500];
```